gradle/java-library.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'java'
+apply plugin: 'checkstyle'
```

Comment:
@TimvdLippe, we don't need any extra dependency. checkstyle plugin is a part of Gradle distribution. Good question!